docker/px4-dev/scripts/archlinux_install_script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
@jlecoeur Original docs additionally installed these: lib32-glibc python-pyserial python-numpy  toml - are they not needed on archlinux?

Reply:
@hamishwillee not for what I tested at least.
I has been in the doc for a long time: https://github.com/PX4/Devguide/commit/b2131f18f35d5cbd87182f21b2d5681c94937877#diff-1e96ef0fe3c2abb013c67b2245857406
Maybe it is not required anymore?

Reply:
Thanks. @dagar , any idea whether  lib32-glibc  is needed in builds? / What is it used for?

Reply:
Probably for the 32bit version of the arm toolchain. The next time we update it will be 64bit linux. 

![image](https://user-images.githubusercontent.com/84712/33741097-0745e81c-db71-11e7-8cc8-45b9a5c7f0be.png)